src/main/java/online/shop/service/ProductServiceImpl.java

```diff
@@ -0,0 +1,39 @@
+package online.shop.service;
```

Comment:
I think you might be asked to create a package "impl" in the package "service" and move the implementations there.